build.assets/Makefile

```diff
@@ -72,7 +75,14 @@ TELEKUBE_TSH_TAG := v3.2.13
 # Extra flags that may be provided when publishing telekube artifacts (e.g. --insecure)
 TELEKUBE_PUBLISH_FLAGS ?=
 
+GOCACHE_DOCKER_OPTIONS ?=
+GOCACHE ?= $(HOME)/.cache/go-build
```

Comment:
This helps tremendously during development and it's off by default (hence should not be an issue on jenkins) so I'm able to opt-in if necessary.